Source/HTTP/Curl/CurlEasyRequest.h

```diff
@@ -7,7 +7,11 @@
 #else
 // This path is untested, but this http provider should work with other curl implementations as well.
 // The logic in CurlMulti::Perform is optimized for XCurl, but should work on any curl implementation.
-#include <curl.h>
+#include <curl/curl.h>
+//The extern structs are here for LibHC Linux. CurlEasyRequest, CurlProvider, and CurlMulti create a circular dependency.
```

Comment:
1) can you expand on what the issue is?
2) can these declarations be moved into cpp files?
3) you should use HC_DECLARE_TRACE_AREA and not name the globals directly

Reply:
1. John explained to me what the problem was a while back and it had to do with how the compilers handled the declaration of traces in the .h file. The Clang Compiler doesn't know what should be filled in for g_trace while MSBuild did. 
2. Tried putting them in the cpp files and that did not work.
3. I should do that.